src/combinatorics/simple_graph/basic.lean

```diff
@@ -281,9 +281,55 @@ def edge_set : set (sym2 V) := sym2.from_rel G.symm
 
 @[simp] lemma mem_edge_set : ⟦(v, w)⟧ ∈ G.edge_set ↔ G.adj v w := iff.rfl
 
+lemma not_is_diag_of_mem_edge_set {e : sym2 V} (h : e ∈ G.edge_set) : ¬ e.is_diag :=
+sym2.ind (λ v w, adj.ne) e h
+
+@[simp] lemma edge_set_sdiff (G H : simple_graph V) :
+  (G \ H).edge_set = G.edge_set \ H.edge_set :=
+by { ext ⟨x,y⟩, exact ⟨id, id⟩ }
```

Comment:
In increasing order of golf (not sure all work!),
```suggestion
by { ext ⟨x, y⟩, refl }
```
Same for the other ones.
```suggestion
by { ext, refl }
```
```suggestion
set.ext $ λ _, rfl
```
```suggestion
rfl
```

Reply:
The first does works